gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -293,27 +320,30 @@ public class GoogleHadoopFileSystemConfiguration {
   public static final HadoopConfigurationProperty<Boolean>
       GCS_INPUT_STREAM_FAST_FAIL_ON_NOT_FOUND_ENABLE =
           new HadoopConfigurationProperty<>(
-              "fs.gs.inputstream.fast.fail.on.not.found.enable", true);
+              "fs.gs.inputstream.fast.fail.on.not.found.enable",
+              DEFAULT_FAST_FAIL_ON_NOT_FOUND_ENABLED);
 
   /**
-   * If true, reading a file with GZIP content encoding (HTTP header "Content-Encoding: gzip") will
+   * If false, reading a file with GZIP content encoding (HTTP header "Content-Encoding: gzip") will
    * result in failure (IOException is thrown).
    */
   public static final HadoopConfigurationProperty<Boolean>
       GCS_INPUT_STREAM_SUPPORT_GZIP_ENCODING_ENABLE =
           new HadoopConfigurationProperty<>(
-              "fs.gs.inputstream.support.gzip.encoding.enable", false);
+              "fs.gs.inputstream.support.gzip.encoding.enable",
+              DEFAULT_GZIP_ENCODING_SUPPORT_ENABLED);
```

Comment:
@medb  could you please look into this - I have enabled gzip by default. Previously, it was disabled here but was enabled in gcsio. Does it look correct?

Reply:
Disabled it by default now.